common/lib/capa/capa/inputtypes.py

```diff
@@ -325,12 +326,15 @@ def _get_render_context(self):
             'describedby_html': '',
         }
 
-        # Don't add aria-describedby attribute if there are no descriptions
-        if self.response_data.get('descriptions'):
-            description_ids = ' '.join(self.response_data.get('descriptions').keys())
-            context.update(
-                {'describedby_html': 'aria-describedby="{}"'.format(description_ids)}
-            )
+        # Generate the list of ids to be used with the aria-describedby field.
+        # Every list should contain the status id
+        status_id = 'status_' + self.input_id
+        descriptions = list([status_id])
+        descriptions.extend(self.response_data.get('descriptions', {}).keys())
+        description_ids = ' '.join(descriptions)
+        context.update(
+            {'describedby_html': 'aria-describedby="{}"'.format(description_ids)}
```

Comment:
Usually the suffix `_html` implies that this is HTML and is HTML ready.  Assuming that is what you want, you probably want:
```
{'describedby_html': HTML('aria-describedby="{}"').format(description_ids)}
```
This will HTML-escape the describedby in case there is any html in there.

I still need to continue this on Monday...